src/android/toga_android/widgets/selection.py

```diff
@@ -22,9 +22,6 @@ def create(self):
         self.native.setOnItemSelectedListener(TogaOnItemSelectedListener(
             impl=self
         ))
-        # On Android, the list of options is provided to the `Spinner` wrapped in
-        # an `ArrayAdapter`. We store `self.adapter` to avoid having to typecast it
-        # in `add_item()`.
```

Comment:
This comment explains why we don't call `self.native.getAdapter`. But it no longer applies for the same reason as above: Rubicon return would return the declared type `SpinnerAdapter`, which would need to be casted in order to modify it,  but Chaquopy would return the actual type `ArrayAdapter`.